src/core/manager.c

```diff
@@ -2094,13 +2094,15 @@ void manager_dump(Manager *m, FILE *f, const char *prefix) {
         assert(f);
 
         for (q = 0; q < _MANAGER_TIMESTAMP_MAX; q++) {
+                const dual_timestamp *t = m->timestamps + q;
                 char buf[FORMAT_TIMESTAMP_MAX];
```

Comment:
Please update the size of the buffer.
```
char buf[FORMAT_TIMESPAN_MAX];
assert_cc(FORMAT_TIMESPAN_MAX >= FORMAT_TIMESTAMP_MAX);
```

Reply:
or maybe `char buf[CONST_MAX(FORMAT_TIMESPAN_MAX, FORMAT_TIMESTAMP_MAX)];`

Reply:
Yep, good idea. Updated.